.changeset/three-emus-tell.md

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
A general comment, this changeset is very long. In the future, should these long-formed / detailed changelogs being documented on the site and referenced from changeset

Reply:
We can always blog on changes, slightly worried about relying on the site too much, as the site is subject to change, whilst a changelog is immutable.